tests/resources/VCH-Util.robot

```diff
@@ -76,11 +76,16 @@ Set Test Environment Variables
     Set Test VCH Name
     # cleanup any potential old certs directories
     Remove Directory  %{VCH-NAME}  recursive=${true}
+    Wait Until Keyword Succeeds  5x  1s  Create Unique Bridge Network
+
+Create Unique Bridge Network
     # Set a unique bridge network for each VCH that has a random VLAN ID
     ${vlan}=  Evaluate  str(random.randint(1, 4093))  modules=random
-    ${out}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc host.portgroup.add -vlan=${vlan} -vswitch vSwitchLAN %{VCH-NAME}-bridge
-    ${out}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Run  govc dvs.portgroup.add -vlan=${vlan} -dvs test-ds %{VCH-NAME}-bridge
-    Set Environment Variable  BRIDGE_NETWORK  %{VCH-NAME}-bridge
+    ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run And Return Rc And Output  govc host.portgroup.add -vlan=${vlan} -vswitch vSwitchLAN %{VCH-NAME}-${vlan}
+    Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Run And Return Rc And Output  govc dvs.portgroup.add -vlan=${vlan} -dvs test-ds %{VCH-NAME}-${vlan}
```

Comment:
DPG should **not** have the VCH name in it. If it does then we do not benefit from any name based reservation of the VLAN id.

Reply:
Sure I can continue to tweak this, but you are seriously over engineering a non-existent problem.  The *only* case in which this PR logic kicks in is if within a single build we fail to cleanup previous VCHs and bridges on a previous test *and then* we happen to get the same random vlan number as one of those previously not cleaned up bridges. 

Changing to an indefinite until keyword succeeds will just mask that further until we consume all 4096 vlans.  And we cannot remove the VCH name from the bridge name otherwise we have *no* simple way to find and cleanup leaked bridge networks.